src/thunderbots/software/util/logger/custom_logging_levels.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <g3log/loglevels.hpp>
+
+// See https://github.com/KjellKod/g3log/blob/master/API.markdown#custom-logging-levels
+// for adding custom log levels
+
+// A custom level used to specify messages that contain robot statuses, including various
+// diagnostic information like battery voltage and wheel encoder errors. These messages
+// are separate because they are shown separately in the Visualizer.
+// over radio
+const LEVELS ROBOT_STATUS{INFO.value + 1, {"ROBOT_STATUS"}};
```

Comment:
Is `INFO.value + 1` really the best way to do this? Seems like a bit of a hack?

Reply:
No, this is how log levels are typically added. Each log level has its own value, which affects how it's handled (eg. levels >= WARNING print to `stderr`, and levels >= FATAL will abort the system).

This values could _technically_ be the same as `INFO` and work okay, but I think this makes it clearer that the levels are distinct. This is explained in the link in the comment